tendrl/node_agent/flows/import_cluster/__init__.py

```diff
@@ -21,25 +22,23 @@ def run(self):
         integration_id = self.parameters['TendrlContext.integration_id']
         tendrl_ns.tendrl_context.integration_id = integration_id
         tendrl_ns.tendrl_context.save()
-        node_list = self.parameters['Node[]']
+        node_list = self.parameters['Node[]'].split(", ")
```

Comment:
Node[] is command separated list now? Have you published a new format of job structure to API team? I feel a good comment on this PR would help and tag @anivargi for this.